docs/how-to/installation/docker.txt

```diff
@@ -7,41 +7,73 @@ Preliminary Setup
 -----------------
 
 #. :ref:`Install Docker <install-docker>` on all machines in the
-   cluster. If the agents have GPUs, ensure that the :ref:`Nvidia
+   cluster. If the agent instances have GPUs, ensure that the :ref:`Nvidia
    Container Toolkit <validate-nvidia-container-toolkit>` on each one is
    working as expected.
 
-#. Pull the official Docker images for PostgreSQL. We
-   recommend using the version listed below.
+#. Pull the official Docker image for PostgreSQL. We recommend using the
+   version listed below.
 
    .. code::
 
       docker pull postgres:10
 
-   This image is not provided by Determined AI; please see its
-   respective Docker Hub page (`PostgreSQL
-   <https://hub.docker.com/_/postgres>`_) for more
-   information.
+   This image is not provided by Determined AI; please see `its Docker Hub page
+   <https://hub.docker.com/_/postgres>`_ for more information.
 
-#. Pull the Docker image for the master or agent on each machine
-   (replace ``VERSION`` with a valid Determined version, such as the
-   current version, |version|):
+#. Pull the Docker image for the master or agent on each instance where these
+   services will run. There is a single master container running in a
+   :ref:`Determined cluster <det-system-architecture>`, and typically there is
+   one agent container per compute instance. A single instance can host both the
+   master container and an agent container. Replace ``VERSION`` with a valid
+   Determined version, such as the current version, |version|:
 
    .. code::
 
       docker pull determinedai/determined-master:VERSION
       docker pull determinedai/determined-agent:VERSION
 
+#. Assuming you are running multiple containerized services on the Determined
+   master instance, optionally create the ``determined`` network that we will
+   use in the service startup commands below:
+
+   .. code::
+
+      docker network create determined
+
+
 Configuring and Starting the Cluster
 ------------------------------------
 
-Determined Master and Agents
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+PostgreSQL
+~~~~~~~~~~
+The following command starts the PostgreSQL container on the master instance
+using the ``determined`` network created above:
 
-:ref:`Configuration values <cluster-configuration>` can come from a
-file, environment variables, or command-line arguments. To get a default
-configuration file for the master, which contains a listing of the
-available options and descriptions for them, run:
+.. code::
+
+   docker run \
+       --name determined-db \
+       --network determined \
+       -p 5432:5432 \
+       -v determined_db:/var/lib/postgresql/data \
+       -e POSTGRES_DB=determined \
+       -e POSTGRES_PASSWORD=<DB password> \
+       postgres:10
+
+If the master container will connect to PostgreSQL via Docker networking,
```

Comment:
non-blocking: I'm not sure "container" is necessary all of the times it shows up; the distinction between the program and the container is smaller and less important than the distinction between them and the machine, and it gets a bit repetitive having them all there.

(Also, in this and some other cases, not having "container" would sound strictly better; here, for example, it's not the container making the connection so much as the program inside it.)

Reply:
Got rid of a few superfluous container / instance mentions